kuma/users/checks.py

```diff
@@ -2,43 +2,48 @@
 from django.conf import settings
 from django.core.checks import Error
 
-from .stripe_utils import create_missing_stripe_webhook
+from .stripe_utils import create_missing_stripe_webhook, retrieve_stripe_prices
 
-STRIPE_PLAN_ERROR = "kuma.users.E001"
-STRIPE_PLAN_INACTIVE_ERROR = "kuma.users.E002"
+STRIPE_ERROR = "kuma.users.E001"
+STRIPE_PRICES_MISSING_ERROR = "kuma.users.E002"
+STRIPE_PRICE_INACTIVE_ERROR = "kuma.users.E003"
+STRIPE_PRICE_ERROR = "kuma.users.E004"
 
 
 def stripe_check(app_configs, **kwargs):
     errors = []
 
-    if not (
-        settings.STRIPE_SECRET_KEY
-        or settings.STRIPE_PUBLIC_KEY
-        or settings.STRIPE_PLAN_ID
-    ):
+    if not settings.STRIPE_SECRET_KEY:
         return errors
 
     try:
-        plan = stripe.Plan.retrieve(settings.STRIPE_PLAN_ID)
-        if not plan.active:
+        prices = retrieve_stripe_prices()
+        if not prices:
```

Comment:
I like this! If you have set `STRIPE_SECRET_KEY` you *have* to have set prices too.